bridgetown-core/lib/bridgetown-core/collection.rb

```diff
@@ -279,10 +279,15 @@ def read_resource(full_path, manifest: nil)
       )
 
       if model_is_multi_locale?(model, model_relative_path)
-        site.config.available_locales.each do |locale|
+        # If the model specifies a locales key, use that to determine the
+        # the locale of each resource, otherwise fall back to `site.config.available_locales`
+        locales = model.locales || site.config.available_locales
+
+        locales.map do |locale|
           model.locale = locale
           add_resource_from_model model
         end
+
         return
```

Comment:
Unrelated to this PR @jaredcwhite - the return value of this method is inconsistent. When adding a multi-locale model, it returns nil, but when it doesn't have multi, then it returns the output of `add_resource_from_model model`.

This may not be a bug, and I tried to trace it back to see what the effects were, but I couldn't quite follow. Just highlighting this since I noticed it.

I'm realizing now that I left the `locales.map` when I meant to make it `locales.each`, but... maybe this should change to:

```ruby
return locales.map do |locale|
  model.locale = locale
  add_resource_from_model model
end
```

Reply:
@rickychilcott I think the return value of this method can be void, so fine with refactoring to do that and adding a `@return [void]` to the comment doc.

Reply:
👍